.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [stable, beta, nightly]
+
+    env:
+      RUSTFLAGS: -D warnings
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        rustup set profile minimal
+        rustup install ${{ matrix.rust }}
+        rustup default ${{ matrix.rust }}
+
+        cargo build --features v2_6_32
+        cargo build --features v3_2
+        cargo build --features v3_10
+        cargo build --features v4_2
+        cargo build --features v4_4
+        cargo build --features v4_20
+        cargo build --features v5_4
+        cargo build --features v5_11
+        cargo build --features "v3_2 netlink"
```

Comment:
To save some CI time, could these be "check" instead of "build"?

Reply:
Each individual build takes less than 2s though, so it doesn't save much. In addition there is barely any codegen that can be skipped using "check". If you prefer I can switch though.

Reply:
That's true; leaving it as cargo build sounds fine.

Reply:
Switched to cargo check.